### ISSUE TITLE
[Feature Request] pf run create add overwrite feature

### ISSUE THREAD

**Issue by sashokbg** [microsoft/promptflow#2448]
**Is your feature request related to a problem? Please describe.**

During early development stage it is useful to just replace the existing run and overwrite it.
Currently I am forced to do ```pf run list``` and/or ```pf run delete -n <name>```

PS: This is probably low priority :)

**Describe the solution you'd like**
I propose adding a --overwrite arg to the```pf run create``` command that will delete the existing run if any.

**Additional Context**

My current workflow is like so:

```
pf run delete -n first_run -y
pf run delete -n first_run_eval -y
pf run create --flow . --data batch_data.jsonl --stream --name first_run --column-mapping question='${data.question}'
pf run create --flow evaluation --data batch_data.jsonl --column-mapping groundtruth_link='${data.link}' --column-mapping question='${data.question}'  prediction='${run.outputs.answer}' --run first_run --stream --name first_run_eval
```

**Comment by 0mza987** [comment]
Hi @sashokbg,

Batch run record is designed to be a matter of fact and represents a certain stage of the developed flow, so we don't expect users to always create a batch run with the same run name.

For early development stage, we recommend to use `pf flow test` to debug the flow, and only create a batch run when the flow is improved enough and you want to run it against a large size of data.

**Comment by sashokbg** [comment]
Hello @0mza987 fair enough, I ended writing a small SH script that runs ```pf run delete -n ${run_name}``` before starting.
Thank you for your feedback !
